### ISSUE TITLE
Typescript typings are missing the useStateLinkUnmounted transform parameter

### ISSUE THREAD

**Issue by Lani** [avkonst/hookstate#10]
Hi,

I think that the overloads for [useStateLinkUnmounted](https://github.com/avkonst/hookstate/blob/efef132cc8c5f8102332634da8feec984277758a/dist/UseStateLink.d.ts#L51) with the transform parameter are missing.

_This library looks really promising. I like the api and the flexibility with global, scoped and local states. :)_

**Comment by avkonst** [comment]
Hi, thanks for your feedback. I appreciate if you help it to grow by spreading some words about it. Thanks.

Regarding the missing parameter. I am looking a t it now. Could you please confirm which variant you are using with StateInf or StateRef? If you could provide some code snippet you are working with, I could recommend on the best way to apply the function.

**Comment by avkonst** [comment]
I think this one you found missing.
```ts
export function useStateLinkUnmounted<S, R>(
    source: StateRef<S>,
    transform?: (state: StateLink<S>) => R
): StateLink<S>;
```

**Comment by avkonst** [comment]
Fixed in 1.1.3
